web/src/pages/apd/activities/contractor-costs/ContractorResources.cy.js

```diff
@@ -64,5 +75,44 @@ describe('<ContractorResources>', () => {
         }
       }
     });
+    cy.contains('Private Contractor Costs').should('exist');
+
+    // Getting an uncaught:exception on save:
+    // TypeError: Cannot destructure property 'keyPersonnel' of 'bigState.apd.data.keyStatePersonnel' as it is undefined.
```

Comment:
Is there a way you can provide a keyPersonnel? Tif mentioned yesterday that the budget calculation is likely running so you may need to provide some minimum pieces of state to satisfy the budget calculation running.